logs_api_int_script.py

```diff
@@ -204,26 +190,57 @@ def insert_data_to_prod(db, table):
 
     get_clickhouse_data(q)
 
+def load_logs_api_data(api_key, date1, date2, token):
+    url_tmpl = 'https://api.appmetrica.yandex.ru/logs/v1/export/events.csv?application_id={api_key}&date_since={date1}%2000%3A00%3A00&date_until={date2}%2023%3A59%3A59&date_dimension=default&fields=event_name%2Cevent_timestamp%2Cappmetrica_device_id%2Cos_name%2Ccountry_iso_code%2Ccity&oauth_token={token}'
+    url = url_tmpl.format(api_key=api_key, date1=date1, date2=date2, token=token)
 
-def process_date(date, token, api_key, db, table):
-    df = load_logs_api_data(api_key, date, date, token)
-    df = df.drop_duplicates()
-    df['api_key'] = api_key
+    r = requests.get(url)
+
+    while r.status_code != 200:
+        print r.status_code,
+        if r.status_code != 202:
+            raise ValueError, r.text
 
-    drop_table('default', 'tmp_data')
-    drop_table('default', 'tmp_data_ins')
+        time.sleep(10)
+        r = requests.get(url)
 
-    table_create('default', 'tmp_data')
+    resp_rows = r.text.split('\n')
```

Comment:
The whole response will be loaded here and it can lead memory limitation breaking